stylelint-config-landr/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "stylelint-config-landr",
```

Comment:
I'd like to see all our packages unscoped as that seems to be the trend when it comes to these open source packages.
- [eslint-config-airbnb](https://www.npmjs.com/package/eslint-config-airbnb)
- [eslint-config-google](https://github.com/google/eslint-config-google)
- [eslint-config-fbjs](https://www.npmjs.com/package/eslint-config-fbjs)

And following that naming convention, the stylelint package should be named `stylelint-config-landr` and the [prettier package](https://github.com/Mixgenius/linting-and-formatting/pull/6) should be named `prettier-config-landr`.

Reply:
I'm also not sure if scoped packages work for stylelint since there is no mention of scoped packages in [their documentation](https://stylelint.io/user-guide/configuration/).

> Because of extends, you can create and use shareable stylelint configurations. Use the stylelint-config keyword within your package.json if publishing your config to npm.

I know for sure that `stylelint-config-landr` works, so I don't see why we should change it.